src/bin/wasm-tools/component.rs

```diff
@@ -318,7 +323,21 @@ impl EmbedOpts {
     /// Executes the application.
     fn run(self) -> Result<()> {
         let (resolve, pkg_id) = self.resolve.load()?;
+
         let world = resolve.select_world(pkg_id, self.world.as_deref())?;
+
+        if self.only_custom {
+            let encoded = metadata::encode(
+                &resolve,
+                world,
+                self.encoding.unwrap_or(StringEncoding::UTF8),
+                None,
+            )?;
+
+            self.io.output_wasm(&encoded, self.wat)?;
```

Comment:
Since this isn't wasm I think that `false` should be passed here instead of `self.wat`?